### DISCUSSION TITLE
赶紧插点火把防止生成骷髅

### DISCUSSION THREAD

**Discussion by hite385** [share121/inter-knot#1908] [其他]
![1722682681276](https://github.com/user-attachments/assets/044923b9-5d2f-48d2-894a-3ff6cab2ad03)

**Comment by YWphosphenes02** [comment]
嘟嘟哒嘟哒嘟

**Comment by AnbulesPeo** [comment]
嘟嘟哒嘟嘟哒

**Comment by lllliufeng** [comment]
晚辣

**Comment by cpu644** [comment]
嘟嘟哒嘟嘟哒

**Comment by inkluv** [comment]
你说得对，但是《我的世界》（Minecraft）是一款沙盒类电子游戏，开创者为马库斯·阿列克谢·泊松（Notch）。游戏由Mojang Studios维护，现隶属于微软Xbox游戏工作室。游戏最初于2009年5月17日作为Classic版本发布，并于2011年11月18日发布Java正式版。我的世界的游戏平台囊括桌面设备、移动设备和游戏主机。中国版现由网易游戏代理[7]，于2016年5月20日在中国大陆运营[34]。

**Comment by 1451374815** [comment]
kala

**Comment by 1luik** [comment]
![46733](https://github.com/user-attachments/assets/d2bba2b3-e22a-4711-b795-6d4e54fb5274)
